website/templates/public/pages/faq.mako

```diff
@@ -118,7 +118,7 @@
 <p>If the project is private, claim the account, and then follow the steps above.</p>
 
 <h3>How do I make folders to organize the files in my project?</h3>
-<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
+<p>You can create a folder or a component to help organize your files. To create a folder, navigate to the files tab of your project. Find the storage add-on you'd like to put a folder in and click the "+" sign next to its name. Components also act like folders, but can be used to control privacy settings independently of the whole project. To create a component, navigate to the project overview page and click “create component” on the right half of the page.</p>
```

Comment:
I can't quite tell what question this FAQ is answering.  It partly seems to be talking about managing projects/components and partly about putting folders in add-ons.  Does the latter require an FAQ?  And, probably shouldn't be mixed right?  Perhaps I am just misunderstanding what the text is trying to explain.